### PR TITLE
Why are Private API Gateways so hard?

### PR DESCRIPTION
**Pull request by Chriscbr**

### CODE REVIEW

**Comment by Chriscbr** [comment]
Needs a link to the tutorial post

File: blog/2024-02-15-private-api-gateways.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+In this post, we’re going to deep dive on the private API gateway pattern: including what a private API gateway entails, its distinct advantages, and why you or your team might opt for this route over other services.
+
+Based on our team’s experience, we’ll be focusing on the capabilities available when building private API gateways on AWS using their managed [VPC](https://docs.aws.amazon.com/vpc/latest/userguide/what-is-amazon-vpc.html) and [API Gateway](https://docs.aws.amazon.com/apigateway/latest/developerguide/welcome.html) services. But many of the lessons will also apply to cloud applications built using Azure’s [API Management service](https://learn.microsoft.com/en-us/azure/api-management/private-endpoint) and other major cloud providers. 
+
+## Understanding Private API Gateways
+
+If we simplify what a private API gateway is, it's a secure means of exposing a set of APIs within a private network, typically established using a [Virtual Private Cloud](https://en.wikipedia.org/wiki/Virtual_private_cloud) (VPC). Let’s first understand each of these.
```
Comment (revitalbarletz) [inline blog/2024-02-15-private-api-gateways.md:19]: If we simply explain/describe?

**Comment by revitalbarletz** [comment]
@Chriscbr  this is great!

File: blog/2024-02-15-private-api-gateways.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
+---
+title: Understanding Private API Gateways
```
Comment (ShaiBer) [inline blog/2024-02-15-private-api-gateways.md:2]: This is the title that was liked best when we discussed how to syndicate it

```suggestion
title: Why are Private API Gateways so hard?
```

Comment (Chriscbr) [reply blog/2024-02-15-private-api-gateways.md:2]: Sounds good 👍

File: blog/2024-02-15-private-api-gateways.md (hunk 1 above)
Comment (Chriscbr) [reply blog/2024-02-15-private-api-gateways.md:19]: Not sure I understand the comment - happy to reword the sentence / paragraph if you have suggestions

**Pull request merged**